pyanaconda/modules/payloads/payload/payload_base_interface.py

```diff
@@ -78,6 +78,9 @@ def DefaultSourceType(self) -> Str:
 
         :return: a string representation of a source type
         """
+        if self.implementation.default_source_type is None:
```

Comment:
Shouldn't we at least log a warning if we do get there & add a FIXME docstring if this really is temporary ?